projects/packages/videopress/package.json

```diff
@@ -80,6 +80,7 @@
 		"@wordpress/icons": "9.13.0",
 		"@wordpress/url": "3.23.0",
 		"classnames": "2.3.1",
+		"copy-webpack-plugin": "11.0.0",
```

Comment:
[nitpick] This should probably go in the devDependencies, along with other packages that are used for the build rather than the functionality. Not that it makes a huge difference.